src/components/MarketDetail/index.js

```diff
@@ -104,7 +105,25 @@ class MarketDetail extends Component {
       marketFetchError: undefined,
     }
   }
+
   componentWillMount() {
+    if (!this.props.params.view) {
+      this.props.changeUrl(`/markets/${this.props.params.id}/${DEFAULT_VIEW}`)
+    }
+
+    this.fetchEssentialData()
+    this.props.requestGasCost(GAS_COST.BUY_SHARES)
+    this.props.requestGasCost(GAS_COST.SELL_SHARES)
+
+    this._fetchDataTimer = setInterval(this.fetchEssentialData, config.fetchMarketTimeInterval)
```

Comment:
This is a linter error, no? [`no-underscore-dangle`](https://eslint.org/docs/rules/no-underscore-dangle)

Reply:
@andre-meyer Yep, it's there. I wasn't sure should I use underscore or no, but then I saw that its used in another files, so I used it. I'm not sure if we should add new eslint rule or not to use underscore

Reply:
Really? I'd like to avoid changing the linter rules and just avoid underscores for now